src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -585,7 +588,7 @@ public void buttonAddCvrFileClicked() {
             getTextOrEmptyString(textFieldCvrIdCol),
             getTextOrEmptyString(textFieldCvrPrecinctCol),
             getTextOrEmptyString(textFieldCvrOvervoteDelimiter),
-            getProviderChoice(choiceCvrProvider).toString(),
+            getProviderChoice(choiceCvrProvider).getInternalLabel(),
```

Comment:
Agree that it'd be nice to make this display the GUI label instead, and also disable the field for editing (or even all rows in the CVR table in general).

Reply:
will disable all editing

Reply:
Sounds good, please note this clearly in the squash commit log!